docs/advanced/subclassed-layers.md

```diff
@@ -59,17 +59,22 @@ export default MultiColorPathLayer extends PathLayer {
 
 You can replace the shaders used in a layer by overriding the `getShaders()`
```

Comment:
luma.gl's shader module documentation talks more about `assemberShaders` and `Model` class,  not about shader modules in general, probably not a good link.